site/en/docs/extensions/mv3/network-requests/index.md

```diff
@@ -73,15 +73,15 @@ non-secure HTTP access to a given host or set of hosts, it must declare the perm
 
 ## Fetch() vs. XMLHttpRequest()
 
-`fetch()` was created specifically for service workers and follows a broader web trend away from synchronous operations. The `XMLHttpRequest` API is supported in extensions outside of the service worker, and calling `XMLHttpRequest` will trigger the service worker's fetch handler. New work should favor the use of `fetch()` wherever possible. 
+`fetch()` was created specifically for service workers and follows a broader web trend away from synchronous operations. The `XMLHttpRequest()` API is supported in extensions outside of the service worker, and calling it triggers the extension service worker's fetch handler. New work should favor `fetch()` wherever possible. 
 
 ## Security considerations {: #security-considerations }
 
-### Avoiding cross-site scripting vulnerabilities {: #xss }
+### Avoid cross-site scripting vulnerabilities {: #xss }
 
 When using resources retrieved via `fetch()`, your extension service worker should be careful not to
-fall victim to [cross-site scripting][9]. Specifically, avoid using dangerous APIs such as the
-below:
+fall victim to [cross-site scripting][9]. Specifically, avoid using dangerous APIs such as
+`innerHTML` or `eval()`. For example:
```

Comment:
Nit: I probably wouldn't mention eval here given it isn't possible in MV3.

Reply:
I didn't say "don't use it". I said "avoid it". Isn't that valid?

Reply:
I misread your statement.

Reply:
Done.